### PR TITLE
feat(bigquery): implement get and save method for integration with API

### PR DESCRIPTION
**Pull request by kapoc3**
implemnted methods for integration with backend, temporal change the efect when response ok or failure

imagen

imagen

### CODE REVIEW

File: src/components/Integrations/BigQuery/ControlPanel/AuthorizationPage.js [hunk 1]
```diff
@@ -22,13 +21,13 @@ const AuthorizationLayout = ({ dependencies: { bigQueryClient } }) => {
...
 
   const onSubmit = async (values) => {
-    // TODO: integrate actual implementation to save
-    await new Promise((resolve) => {
-      createTimeout(() => {
-        resolve(true);
-        alert('Sent: ' + JSON.stringify([...values.emails]));
-      }, 100);
-    });
+    const parameter = { emails: [...values.emails] };
```
Comment (jcamposmk) [inline src/components/Integrations/BigQuery/ControlPanel/AuthorizationPage.js:24]:
```suggestion 
const emailsData = { emails: values.emails };
```

**Comment by jcamposmk** [comment]
I don't see related test to this task.

**Comment by cbernat** [comment]
@kapoc3 could you add an entry in new features to access big query implementation more easy? We have all incomplete features accesible through here -> https://cdn.fromdoppler.com/doppler-webapp/int-build3709/#/new-features

**Comment by cbernat** [comment]
You should consider adding `cdn.fromdoppler.com` as a CORS available domain inside your api for testing
image

**Comment by kapoc3** [comment]
> 
> 
> @kapoc3 could you add an entry in new features to access big query implementation more easy? We have all incomplete features accesible through here -> https://cdn.fromdoppler.com/doppler-webapp/int-build3709/#/new-features

this change will be in a new PR

**Comment by cbernat** [comment]
> You should consider adding `cdn.fromdoppler.com` as a CORS available domain inside your api for testing
> image

What about this @kapoc3 ?

**Comment by kapoc3** [comment]
> 
> 
> I don't see related test to this task.

the test of this implementation can be found in AuthorizationForm.test.js

**Pull request merged**